backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -665,14 +676,64 @@ public static List<FeedbackMessage> validatePositiveHIVRequiredAOEFields(
               .scope(ITEM_SCOPE)
               .fieldHeader(pregnant.getHeader())
               .source(ResultUploadErrorSource.SIMPLE_REPORT)
-              .message(getRequiredHivAoeValuesErrorMessage(pregnant.getHeader()))
+              .message(
+                  getPositiveResultRequiredValueErrorMessage(
+                      pregnant.getHeader(), DiseaseService.HIV_NAME))
               .errorType(ResultUploadErrorType.MISSING_DATA)
               .fieldRequired(pregnant.isRequired())
               .build());
     }
     return errors;
   }
 
+  private static FeedbackMessage buildSyphilisMissingDataFeedbackMessage(ValueOrError field) {
+    return FeedbackMessage.builder()
+        .scope(ITEM_SCOPE)
+        .fieldHeader(field.getHeader())
+        .source(ResultUploadErrorSource.SIMPLE_REPORT)
+        .message(
+            getPositiveResultRequiredValueErrorMessage(
+                field.getHeader(), DiseaseService.SYPHILIS_NAME))
+        .errorType(ResultUploadErrorType.MISSING_DATA)
+        .fieldRequired(field.isRequired())
+        .build();
+  }
+
+  public static List<FeedbackMessage> validatePositiveSyphilisRequiredAOEFields(
+      ValueOrError testResult,
+      ValueOrError gendersOfSexualPartners,
+      ValueOrError pregnant,
+      ValueOrError previousSyphilisDiagnosis,
+      ValueOrError symptomaticForDisease) {
+    List<FeedbackMessage> errors = new ArrayList<>();
+    // includes SNOMED values for positive and detected
+    Set<String> positiveTestResultValues =
+        Set.of(POSITIVE_LITERAL, DETECTED_LITERAL, POSITIVE_CODE, DETECTED_CODE);
+    if (!positiveTestResultValues.contains(testResult.getValue().toLowerCase())) {
+      return errors;
+    }
+
+    if (gendersOfSexualPartners.getValue() == null
```

Comment:
Think we can swap these null check / blank checks with `StringUtils.isBlank(AOEAnswer.getValue())` 

https://stackoverflow.com/questions/23419087/stringutils-isblank-vs-string-isempty